### PR TITLE
bug 1745283 Support file-level tags

### PR DESCRIPTION
**Pull request by chutten**
### Pull Request checklist ###

- [ ] **Quality**: This PR builds and tests run cleanly
  - `make test` runs without emitting any warnings
  - `make lint` runs without emitting any errors
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes a changelog entry to `CHANGELOG.md` or an explanation of why it does not need one
  - Any breaking changes to language binding APIs are noted explicitly

### CODE REVIEW

**Comment by chutten** [comment]
Bit of a draft, this, as I haven't made a CHANGELOG and things. Because, for whatever reason, my local tests aren't picking up my local changes to the metrics schema : (

**Comment by chutten** [comment]
Okay, now could someone explain how my code change resulted in a quotation change in an unrelated test file?

**Comment by chutten** [comment]
Ah, found it. My bad.

File: glean_parser/parser.py [hunk 1]
```diff
@@ -243,6 +245,10 @@ def _instantiate_metrics(
 
             if metric_obj is not None:
                 metric_obj.no_lint = list(set(metric_obj.no_lint + global_no_lint))
+                if len(global_tags) > 0:
```
Comment (badboy) [inline glean_parser/parser.py:248]:
```suggestion
                if len(global_tags):
```

not-zero is truthy (so are empty arrays, but `if len(...)` seems to be used throughout the codebase already)

**Review by badboy — APPROVED** [review]
This should have doc updates in https://mozilla.github.io/glean/book/reference/yaml/metrics.html as well.

**Pull request merged**